next.js/bazel/next_entry.js

```diff
@@ -1,10 +1,15 @@
+// This is a custom next js_binary entry_point.
+// See comment in bazel/next.bzl for more information as to why this is needed.
+
+const path = require('path');
 const { spawnSync } = require('child_process');
-// This path is currently a hack an specific to this projects layout with
-// ../../../../../ being the path from the the next_bin binary runfiles root
-// to the execroot.
-// TODO: Generalize this path in the future.
 const entry = require.resolve(
-  `../../../../../${process.env.BAZEL_BINDIR}/node_modules/next/dist/bin/next`
+  path.join(
```

Comment:
Could this path be replaced with `"$(location %s)" % next_bin` or something like that in the BUILD?

Reply:
`$(location)` would require also passing the target to `data` which I'm avoiding here. This is temporary anyway as I'm going to add a `command` attribute to `js_run_binary`  which will make this custom entry point boil away